src/store/rad.js

```diff
@@ -111,9 +109,15 @@ export default {
         }
         state.currentTemplate.variables = [...state.currentTemplate.variables]
       }
+      this.commit(UPDATE_HISTORY, {
+        mir: state.currentRadonMarkupInterpreter.getMir(),
+      })
```

Comment:
This is out of the scope of this PR and is fixed in #1279

```suggestion
      this.commit(UPDATE_HISTORY, {
        mir: state.currentRadonMarkupInterpreter.getMir(),
      })
```